client/components/repository/common/CreateDialog/index.vue

```diff
@@ -93,6 +100,7 @@ export default {
       this.$emit('created', item);
       this.visible = false;
       this.$router.push({ query: { activityId: item.id } });
+      this.submitting = false;
```

Comment:
In case that the `save` fails, button will be stuck as disabled. We don't handle the error anyway but maybe we should handle setting this flag in the manner of `finally`?

Reply:
This means converting `await this.save` to classic `Promise`. @underscope what do you say?

Reply:
Or using `try/catch/finally`